x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -249,6 +249,8 @@ export const DETECTION_ENGINE_PREPACKAGED_URL =
   `${DETECTION_ENGINE_RULES_URL}/prepackaged` as const;
 export const DETECTION_ENGINE_PRIVILEGES_URL = `${DETECTION_ENGINE_URL}/privileges` as const;
 export const DETECTION_ENGINE_INDEX_URL = `${DETECTION_ENGINE_URL}/index` as const;
+export const DETECTION_ENGINE_SIGNAL_INDEX_URL =
+  `/internal${DETECTION_ENGINE_URL}/signal/index` as const;
```

Comment:
Could you move this to the `Internal detection engine routes` section below and change it to `${INTERNAL_DETECTION_ENGINE_URL}/signal/index`? The other internal routes don't include `/api/` in the URL.

Reply:
https://github.com/elastic/kibana/pull/133708/commits/7f4ac0ef4708547bbb3d696853076c20a1050cea